packages/http-server/src/server.ts

```diff
@@ -77,7 +81,7 @@ export const createServer = (operations: IHttpOperation[], opts: IPrismHttpServe
       body,
     };
 
-    components.logger.info({ input }, 'Request received');
+    components.logger.info({ input: omit(input, 'body') }, 'Request received');
```

Comment:
body could be an octet stream now, so we're not going to log it anymore, because we cannot serialise it to a string.

Well we can, but it's not worth it.

Reply:
I think this was not really interesting anyway — this was logging the body sent by the user — and the user is aware of it anyway I would say.

Reply:
Cool, ping me when you add tests and I can look at this again.

Reply:
We got it